packages/dev/src/declarations.ts

```diff
@@ -62,7 +62,7 @@ type PropertyThemeValue<ThemeKey extends string> =
   | Tokenami.ArbitraryValue
   | CSS.Globals
   | TokensByThemeKey[ThemeKey]
-  | (ThemeKey extends 'grid' ? Tokenami.GridValue : never);
+  | (ThemeKey extends 'grid' | 'number' ? Tokenami.GridValue : never);
```

Comment:
if any properties in the `properties` config have a `['number']` value, they will accept a number type but won't compile the grid calc in stylesheet. i'll document this later but keeping it low-key for now in case i decide to redesign this.